### PR TITLE
set_virtlogd: Add case for test reload virtlogd

### PR DESCRIPTION
**Pull request by cliping**
Start guest with spice graphic, and then reload virtlogd.service.

Signed-off-by: Liping Cheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:
#avocado run --vt-type libvirt --vt-machine-type q35 conf_file.qemu_conf.set_virtlogd.positive_test
JOB ID     : 3fd74dbaf3b2d04a204a70b98999f4af201aa177
JOB LOG    : /root/avocado/job-results/job-2019-07-02T04.51-3fd74db/job.log
 (1/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.default: PASS (114.95 s)
 (2/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.enable_logd: PASS (114.81 s)
 (3/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.reload.vm_with_spice: PASS (115.09 s)
 (4/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.reload.vm_shutoff: PASS (4.49 s)
 (5/5) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.restart: PASS (145.19 s)
RESULTS    : PASS 5 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 496.38 s

File: libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py [hunk 1]
```diff
@@ -161,12 +162,38 @@ def check_service_status(service_name, service_start=False):
             test.fail("Get %s status failed." % service_name)
         return ret.stdout_text.split()[2]
 
+    def reload_and_check_virtlogd():
+        """Reload and check virtlogd status"""
```
Comment (chloerh) [inline libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py]: Maybe split into 3 lines?

File: libvirt/tests/cfg/conf_file/qemu_conf/set_virtlogd.cfg [hunk 2]
```diff
@@ -9,8 +9,12 @@
                 - enable_logd:
                     stdio_handler = "logd"
                 - reload:
-                    start_vm = no
-                    reload_virtlogd = yes
+                    variants:
+                        - vm_with_spice:
+                            with_spice = yes
```
Comment (kylazhang) [inline libvirt/tests/cfg/conf_file/qemu_conf/set_virtlogd.cfg]: If it not related to reload, why it is the sub item of reload?

File: libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py [hunk 3]
```diff
@@ -161,12 +162,42 @@ def check_service_status(service_name, service_start=False):
...
+            test.fail("virtlogd reload failed.")
+        reload_virtlogd_pid = check_service_status("virtlogd", service_start=True)
+        logging.info("After reload, virtlogd PID: %s", reload_virtlogd_pid)
+        if virtlogd_pid != reload_virtlogd_pid:
+            test.fail("After reload, virtlogd PID changed.")
+
+    def configure_spice(vm_name):
+        """
+        Configure spice
+        """
```
Comment (kylazhang) [inline libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py]: missing docstring for params

File: libvirt/tests/cfg/conf_file/qemu_conf/set_virtlogd.cfg (hunk 2 above)
Comment (cliping) [reply libvirt/tests/cfg/conf_file/qemu_conf/set_virtlogd.cfg]: conf_file.qemu_conf.set_virtlogd.positive_test.reload.vm_with_spice: this case need to start guest firstly, and then reload virtlogd (on line 290). 
conf_file.qemu_conf.set_virtlogd.positive_test.reload.vm_shutoff: this case reload virtlogd when guest is shut down.

**Pull request merged**